service/lib/agama/storage/finisher.rb

```diff
@@ -146,7 +146,8 @@ def run?
 
         def run
           glob_files.each do |file|
-            target = File.dirname(file).sub(root_dir, dest_dir)
```

Comment:
Perhaps using `delete_prefix` will communicate better your intention. But as it is already tested, let's keep it as it is.

Reply:
Yeah, already thinking on that, I agree, fixed and also added "/" at the end of the rootdir to show the test was failing with previous implementation but will not change the order of the commits to not force a push.